docs/usage/basic.md

```diff
@@ -4,45 +4,55 @@
+    ```bash
+    IPV4_SUBNET_YOU_EXPECT="172.18.40.0/24"
+    IPV4_IPRANGES_YOU_EXPECT="172.18.40.40-172.20.40.200"
+    ```
 
-```bash
-helm install spiderpool spiderpool/spiderpool --namespace kube-system \
-  --set spiderpoolController.tls.method=auto \
-  --set feature.enableIPv4=true --set feature.enableIPv6=false \
-  --set clusterDefaultPool.installIPv4IPPool=true  \
-  --set clusterDefaultPool.ipv4Subnet=${IPV4_SUBNET_YOU_EXPECT} \
-  --set clusterDefaultPool.ipv4IPRanges={${IPV4_IPRANGES_YOU_EXPECT}}
-```
+3. Deploy Spiderpool with the following command.
 
-Here we disable IPv6, but if you need Spiderpool to work under dual stacks, you can set `feature.enableIPv6=true`. Furthermore, if you also need to create a default IPv6 IPPool, set `clusterDefaultPool.installIPv6IPPool=true` and fill in the desired default IPv6 IPPool information in the `clusterDefaultPool.ipv6Subnet` and `clusterDefaultPool.ipv6IPRanges` parameters at the same time.
+    During the deployment, the necessary TLS certificates and custom resources will be automatically create. See more details about [installation](https://github.com/spidernet-io/spiderpool/blob/main/docs/usage/install.md).
 
-More details about [Spiderpool charts parameters](https://github.com/spidernet-io/spiderpool/blob/main/charts/spiderpool/README.md#parameters).
+    ```bash
+    helm install spiderpool spiderpool/spiderpool --namespace kube-system \
+      --set spiderpoolController.tls.method=auto \
+      --set feature.enableIPv4=true --set feature.enableIPv6=false \
+      --set clusterDefaultPool.installIPv4IPPool=true  \
+      --set clusterDefaultPool.ipv4Subnet=${IPV4_SUBNET_YOU_EXPECT} \
+      --set clusterDefaultPool.ipv4IPRanges={${IPV4_IPRANGES_YOU_EXPECT}}
+    ```
+
+> IPv6 is disabled in this case.
+> 
+> If you want Spiderpool to work under dual stacks, set `feature.enableIPv6=true`.
+> 
+> If you need to create a default IPv6 IPPool, set `clusterDefaultPool.installIPv6IPPool=true` and fill in the desired default IPv6 IPPool information in the `clusterDefaultPool.ipv6Subnet` and `clusterDefaultPool.ipv6IPRanges` parameters at the same time.
+
+See more details about [Spiderpool charts parameters](https://github.com/spidernet-io/spiderpool/blob/main/charts/spiderpool/README.md#parameters).
 
 ## Update CNI network configuration
 
-Then, edit the [CNI network configuration](https://www.cni.dev/docs/spec/#section-1-network-configuration-format) with the default path `/etc/cni/net.d` on the related Nodes so that the Main CNI can use Spiderpool IPAM CNI to allocate IP addresses. Replace the `ipam` object with the following:
+After the install, you should update the [CNI network configuration](https://www.cni.dev/docs/spec/#section-1-network-configuration-format) under the default path `/etc/cni/net.d` on the Node where you want to use Spiderpool, so that the Main CNI can use Spiderpool IPAM CNI to allocate IP addresses.
```

Comment:
```suggestion
After the installation, you should update the
[CNI network configuration](https://www.cni.dev/docs/spec/#section-1-network-configuration-format)
under the default path `/etc/cni/net.d` on the Node where you want to use Spiderpool,
so that the Main CNI can use Spiderpool IPAM CNI to allocate IP addresses.
```